libnetwork/netavark/network.go

```diff
@@ -336,6 +337,39 @@ func (n *netavarkNetwork) DefaultInterfaceName() string {
 	return defaultBridgeName
 }
 
+// NetworkInfo return the network information about binary path,
+// package version and program version.
+func (n *netavarkNetwork) NetworkInfo() types.NetworkInfo {
+	path := n.netavarkBinary
+	packageVersion := cutil.PackageVersion(path)
+	programVersion, err := cutil.ProgramVersion(path)
+	if err != nil {
+		logrus.Warnf("Failed to get netavark version: %v", err)
+	}
```

Comment:
I haven't actually tried but it should be possible to use podman without either cni or netavark installed. Printing this warning by default isn't very friendly to users. Let's lower this to info level.